src/index.js

```diff
@@ -36,7 +44,7 @@ export default class SecureLS {
       config.encodingType.toLowerCase() :
       constants.EncrytionTypes.BASE64;
 
-    this.ls = localStorage;
+    this.ls = root.localStorage || globalStorage;
```

Comment:
Can we please also set the storage type on window?
```
this.ls = root[root.storageType || 'localStorage'] || globalStorage;
```

Reply:
`root` in this context is the window, the aim was to stop the library breaking when used in ssr enabled applications.

I'm not aware of `storageType` as a window property

Reply:
My current work around when using SecureLS in a ssr react app:

```javaScript
const getSecureLS = () => {
  if (typeof window !== 'undefined') {
    return new SecureLS({});
  }
  return {
    set: () => ({}),
    get: () => ({}),
    remove: () => ({}),
  };
};
```